wordpress-mobile/WordPress-Android
# Managing ssl parameter into query for JetPack.

Fixes #11249

Some more context in the inked issue above and in `p1584956107248200-slack-ios-osx-dev`

**NOTE:  I do not have full clarity about the correct procedure to final get merged this PR into the original WP-Utils subtree (after review/approve, let's sync before to merge this one).**

My best understanding is the steps reported here `p1570688351009600-slack-android-dev` but the tests update related to the PR here #11524 AFAIU does not seem to be merged into WPUtils develop (cc @jkmassel in case I'm missing something if he can help to clarify 🙇‍♂️)

### To test:
- Check you can access Media gallery images without errors with:
    - WP.com site, 
    - https JetPack site (added by address or by WP.com login), 
    - http JetPack site (added by address or by WP.com login) 
- Check the WP-Utils connected tests pass
- Check you can repro and get fixed the issue the user had in #11249 setting up Charles Proxy with SSP (further details here `p99K0U-ZF-p2`)

PR submission checklist:
